eng/targets/Services.props

```diff
@@ -48,5 +48,6 @@
   -->
   <ItemGroup>
     <InProcService Include="Microsoft.VisualStudio.LanguageServices.SolutionAssetProvider" />
+    <InProcService Include="Microsoft.VisualStudio.LanguageServices.WorkspaceProjectFactoryService" />
```

Comment:
It's more or less equivalent to [ProvideBrokeredServiceAttribute](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.servicebroker.providebrokeredserviceattribute?view=visualstudiosdk-2022). Rather than using the pkgdef attributes we generate them in MSBuild because CreatePkgDef has issues with cross-plat builds.